renku/cli/dataset.py

```diff
@@ -464,6 +464,10 @@ def edit(short_name, title, description, creator):
         click.echo('Nothing to update.')
     else:
         click.echo('Successfully updated: {}.'.format(', '.join(updated)))
+        if no_email_warnings:
+            click.echo(
+                WARNING + 'No email set for: ' + ', '.join(no_email_warnings)
```

Comment:
maybe it would help here to print the expected syntax on the next line? But perhaps that's too verbose, not sure.

Reply:
I think also it would too verbose; besides, we already print the format in `renku dataset edit -h`.

Reply:
Maybe we can reword slightly then: "No email or wrong format for: "